packages/d3fc-webgl/src/series/glPoint.js

```diff
@@ -87,6 +89,14 @@ export default () => {
         return draw;
     };
 
+    draw.type = (...args) => {
+        if (!args.length) {
+            return type;
+        }
+        type = functor(args[0]);
```

Comment:
Sorry only just spotted that this is where `functor` was being used. Is there any benefit to using it here? Normally it would only be used if it were possible to e.g. change the value per data point.

Reply:
This was only here so that the type could be set per data point in the future, as that functionality isn't added in this PR I will change this